enterprise/static/enterprise/js/enterprise_login_page.js

```diff
@@ -0,0 +1,46 @@
+function redirectToURL(redirectURL) {
+    location.href = redirectURL;
+}
+
+function setupFormSubmit() {
+    $('#enterprise-login-form').submit(function(event){
+        event.preventDefault();
+        var enterpriseSlug = $("#id_enterprise_slug").val();
+
+        $("#activate-progress-icon").removeClass("is-hidden");
+        $("#enterprise-login-submit").attr("disabled", true);
+
+        $.ajax({
+            url : "/enterprise/login",
+            method : "POST",
+
+            beforeSend: function (xhr) {
+                xhr.setRequestHeader("X-CSRFToken", $.cookie("csrftoken"));
+            },
+
+            data : {
+                enterprise_slug : enterpriseSlug
+            },
+
+            success: function(data) {
+                redirectToURL(data.url);
+            },
+
+            error : function(xhr) {
+                $("#activate-progress-icon").addClass("is-hidden");
+                $("#enterprise-login-submit").attr("disabled", false);
+                $("#enterprise-login-form-error")
+                .text(xhr.responseJSON.errors.join(", "))
```

Comment:
Can you show how some of the errors would look to the users?

Reply:
It is same as multiple enterprise selection page.

<img width="669" alt="Screenshot 2020-05-27 at 11 43 27 AM" src="https://user-images.githubusercontent.com/8234192/82986483-589f2600-a00f-11ea-838a-f20b75aa45a0.png">